content/en/tracing/guide/adaptive_sampling.md

```diff
@@ -10,9 +10,6 @@ further_reading:
 - link: "/tracing/trace_pipeline/ingestion_controls"
   tag: "Documentation"
   text: "Ingestion Control Page"
-- link: "/tracing/guide/resource_based_sampling/"
-  tag: "Documentation"
-  text: "Resource-based sampling"
 ---
 
 {{< callout url="<https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/>" d_target="#signupModal" btn_hidden="false" header="Request access to the Preview!" >}}
```

Comment:
Are we going to update this beta page to remove resource-based sampling too?

Reply:
eventually, we should remove this guide and redirect to the ingestion controls page